### ISSUE TITLE
Unable to read message bodies in Lotus Notes

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#543]
**Reported by clev on 2010-02-04 14:35**
Now that it's possible to use NVDA in Lotus Notes (see #80), the message bodies still appear to be inaccessible. When opening a message and trying to read the text using the object navigator plus review keys, I can read only the first line of the text. See attached log. If there is a way to read it without having to select all and copy to notepad, please tell me how. Otherwise I suggest implementing a virtual buffer for it.

**Comment by geoffshang** [comment]
I am using IBM Notes 9.0.1FP5 at work.

I've tested it with NVDA 2016.2.1.

The short answer is that it can read some messages and not others, and same for writing.

Notes has a built-in browser that can be used to read some content types (presumably HTML email).  This browser can be turned off in preferences.  When this browser is used, NVDA is able to read the messages.  It doesn't appear to be possible to get it to use the browser for all email however.

I've not figured out why it is able to read some email composition and not others, presumably it's something similar.

Notes is built on Eclipse, if that helps.

I'm happy to do tests and gather more info if it's of any help.

**Comment by leonardder** [comment]
@geoffshang: Has there anything changed in the situation you reported around a year ago?

**Comment by bhavyashah** [comment]
This is a friendly reminder for @geoffshang  to kindly respond to @leonardder's https://github.com/nvaccess/nvda/issues/543#issuecomment-308225431 by providing a status update for the reported issue.

**Comment by geoffshang** [comment]
@bhavyashah @leonardder Apologies for the delay, I'm fairly neglectful of personal emails 
these days.

Nothing has changed from the NVDA side.  I'm still running the same version of IBM Notes as 
I was in 2016.  I think there have been some point releases since, I need to try updating 
anyway as it crashes with JAWS 2018.  But I'm not expecting this to change much.

**Comment by Adriani90** [comment]
@geoffshang did you already try the newest Lotus Notes build? Dows it work? Or can you still reproduce this issue?

**Comment by bhavyashah** [comment]
This is a friendly reminder for @geoffshang to kindly respond to @Adriani90's https://github.com/nvaccess/nvda/issues/543#issuecomment-405079297 by providing a status update for the reported issue.

**Comment by Adriani90** [comment]
We do not have any updates from the initial author since more than one year. I am closing this issue for now. Please comment on it if you still have this problem and we can reopen the discussion. Thanks.
